step-controller-plugins/step-controller-plugins-timeseries/pom.xml

```diff
@@ -55,6 +55,12 @@
             <version>${step-framework.version}</version>
             <scope>test</scope>
         </dependency>
+         <dependency>
```

Comment:
@lucianbaciu  why do we need this dependency?

Reply:
@david-stephan all tests were failing with an exception coming from that package. after adding it, it worked. now I removed it, but everything still works, so I will clean it. we'll see in the build if something is not ok